AsyncDisplayKit/ASTextNode.mm

```diff
@@ -404,7 +416,7 @@ - (CGSize)calculateSizeThatFits:(CGSize)constrainedSize
 
   [self setNeedsDisplay];
   
-  CGSize size = [self _renderer].size;
+  CGSize size = [self _rendererWithBoundsSlow:{.size=constrainedSize}].size;
```

Comment:
@appleguy Should we add this change in this PR at all? This is just in there for now to have a temporary fix in there that I will remove before we actually merge. We should move that out in to another PR or in case @Adlai-Holler is working on something more awesome we should than just use that.